### DISCUSSION TITLE
Improve Container's Parent-Child naming convention in the Structure Panel / Navigator

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22150] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The new widget container feature in Elementor is very useful, but it has a naming convention issue when inner nested containers are added. Currently, when we add an inner container, it is created with the same name as the parent container, which makes it difficult to distinguish and navigate between them.

![image](https://user-images.githubusercontent.com/99708427/233642626-8b94ad8f-9d98-4272-a9d1-47bb52ca61f4.png)

### Describe the solution you'd like

To improve the user experience and make it easier to navigate nested containers, it is recommended to rename the containers with descriptive names. For example, the parent container could be renamed as "section-container" and the inner containers could be renamed as "row", "col", or "inner-container". Another option would be to use a hierarchical naming convention such as "level 1", "level 2", etc. This would make it easier for users to understand the structure of the containers and navigate them more efficiently.
You can also name Flex container as "Flex" or Grid container as "Grid."

![image](https://user-images.githubusercontent.com/99708427/233644156-b5cb067a-7391-4742-bf94-e865197d534b.png)

### Describe alternatives you've considered

There is no alternative yet. We have to manually rename them.

**Comment by ghost** [comment]
+1

**Comment by ayoubkhan558** [comment]
+1

**Comment by scotreutzel** [comment]
+1
